templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+{{- if .Values.coderd.trustProxyIP -}}
+Deprecated: The coderd "trustProxyIP" setting is deprecated and will
```

Comment:
@khorne3 I didn't spend any time thinking about this really, would love your thoughts on the writing!

Reply:
Maybe

```
The coderd “trustProxyIP” setting is deprecated. Instead, use the coderd “reverseProxy” setting to set trusted headers and
origins. For more information, see: https://coder.com/docs/coder/latest/guides/deployments/proxy
```

Reply:
Thanks Katie! I made the notice more ~~annoying~~ prominent as well.

```
======================= DEPRECATION NOTICE =======================

WARNING: The coderd "trustProxyIP" setting is deprecated. Instead,
use the coderd "reverseProxy" setting to configure trusted headers
and origins.

See https://coder.com/docs/coder/latest/guides/deployments/proxy

======================= DEPRECATION NOTICE =======================
```